src/main/resources/database/changes/release-10.59.0/update_categories_to_use_both_b_and_bi_cases.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE casesvc.category
+SET oldcasesampleunittypes = 'B,BI'
+WHERE categorypk = 'COLLECTION_INSTRUMENT_DOWNLOADED' OR categorypk = 'COLLECTION_INSTRUMENT_ERROR'
+OR categorypk = 'UNSUCCESSFUL_RESPONSE_UPLOADED' OR categorypk = 'SUCCESSFUL_RESPONSE_UPLOADED';
```

Comment:
In the database the PK is UNSUCCESSFUL_RESPONSE_UPLOAD and `SUCCESSFUL_RESPONSE_UPLOAD` so your code wont affect those categories currently